cves/CVE-2015-3982.yml

```diff
@@ -128,8 +135,12 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: |
+    There will need to be a domain specific requirements because session flush can mean anything. In this case its
```

Comment:
I'm confused by this answer. I'm not sure what different session.flush() methods and getting misleading returns has to do with automated testing. Shouldn't an automated test that attempts to login with an empty string as the session key be able to catch this vulnerability?